docs/repositories.md

```diff
@@ -0,0 +1,55 @@
+# Terra Repositories
+
+Terra repositories are Git repositories that contain one or more plugins and bundles. They serve as a central location
+for managing and distributing Terra plugins, allowing users to easily install, update, and manage their plugins through 
+the Terra UI.
+
+## Official Repositories
+
+The official Terra repositories are maintained by the Juno team and contain a collection of plugins and bundles that are
+known to work well together. These repositories are regularly updated with new plugins and bundles, providing users with
+the latest features and improvements.
+
+| Name                   | Maintainer       | Repo URL                                          |
+|------------------------|------------------|---------------------------------------------------|
+| Official Terra Plugins | Juno Innovations | https://github.com/juno-fx/Terra-Official-Plugins |
+
+## Community Repositories
+
+Community repositories are not maintained by the Juno team but are created and managed by the community. These
+repositories can contain plugins and bundles that are not part of the official repositories, allowing users to share
+their own plugins and contributions with the Terra community. Community repositories can be added to Terra in the same
+way as official repositories, but users should exercise caution when installing from non-official sources to ensure the
+plugins are safe and compatible with their environment.
+
+## Repository Structure
+
+### Plugins
+
+Terra repositories are added as "Sources" to Terra via its UI. A Terra repository is a Git repository that contains one 
+or more plugins. Plugins are loaded from the `plugins/` directory in the repository and every Plugin is required to have
```

Comment:
where is the UI? You should link to the genesis docs. Currently we do not have the appStore section, but linking to the genesis section for now would be good.

Reply:
Agreed